### PR TITLE
Update Pagsmile setting

### PR DESCRIPTION
**Pull request by dara123**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by nanobash — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Pagsmile.yaml [hunk 1]
```diff
@@ -16,10 +16,25 @@ allOf:
             description: ID of the Pagsmile application.
           secretKey:
             type: string
-            description: Pagsmile secret key.
+            description: Pagsmile MD5 key.
+            format: password
+            writeOnly: true
+          apiSecretKey:
+            type: string
+            description: Pagsmile API secret key start with Pagsmile_sk.
```
Comment (noon-dawg) [inline openapi/components/schemas/GatewayAccountConfig/Pagsmile.yaml:24]: suggestion: suggestion for readability
```suggestion
            description: Pagsmile API secret key. This key starts with `Pagsmile_sk`.
```

**Review by noon-dawg — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Pagsmile.yaml [hunk 2]
```diff
@@ -16,10 +16,25 @@ allOf:
...
+          apiSecretKey:
+            type: string
+            description: Pagsmile API secret key. This key starts with `Pagsmile_sk`.
             format: password
             writeOnly: true
         required:
           - merchantId
           - appId
           - secretKey
+          - apiSecretKey
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Pagsmile.yaml:31]: Won't this affect existing gateways?

**Comment by dara123** [comment]
@Rebilly/php please review

**Review by ashkarpetin — DISMISSED** [review]

**Review by nanobash — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Pagsmile.yaml [hunk 3]
```diff
@@ -16,10 +16,24 @@ allOf:
             description: ID of the Pagsmile application.
           secretKey:
             type: string
-            description: Pagsmile secret key.
+            description: Pagsmile MD5 key.
+            format: password
+            writeOnly: true
+          apiSecretKey:
+            type: string
+            description: Pagsmile API secret key. This key starts with `Pagsmile_sk`.
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Pagsmile.yaml:24]:
```suggestion
            description: Pagsmile secret key for the Trade Pay API. The key starts with `Pagsmile_sk`.
```
?

**Review by ashkarpetin — APPROVED** [review]

**Review by ninjasimon — APPROVED** [review]

**Review by nanobash — APPROVED** [review]

**Pull request merged**